src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -55,7 +55,10 @@ export const Map: React.FC<MapProps> = ({
   const userUid = user?.uid;
   const showGrid = venue.showGrid;
 
-  const totalColumns = venue.columns ?? DEFAULT_COLUMNS;
+  const totalColumns = Math.max(
+    5,
+    Math.min(100, venue.columns ?? DEFAULT_COLUMNS)
+  );
```

Comment:
This took me a while to mentally process the logic, I think the main thing was that the 'maximum value' was 'before' the 'attempted value'. So my mind read it as MIN MAX venueColumns, rather than `MIN venueColumns MAX`, which feels more natural for a 'range'.

We should also extract the numbers out into `const`s, and probably use those same `const`s in the validation as well as here (which sounds like moving them all into `settings.ts` would be a decent place for them to live.

```suggestion
  const totalColumns = Math.max(
    MINIMUM_COLUMNS,
    Math.min(venue.columns ?? DEFAULT_COLUMNS, MAXIMUM_COLUMNS)
  );
```